docs/guides/weaviate/configuration/using-config-file.md

```
@@ -0,0 +1,226 @@
+title: Run Weaviate with Custom Configuration
+menu:
+  docs_{{ .version }}:
+    identifier: weaviate-using-config-file
+    name: Config File
+    parent: weaviate-configuration
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)